.travis.yml

```diff
@@ -0,0 +1,19 @@
+sudo: false
+language: node_js
+node_js:
+  - stable
+notifications:
+  email: false
+cache:
+  directories:
+    - node_modules
+install:
+  - npm install
+services:
+  - postgresql
+before_script:
+  - psql -c 'drop database if exists barefoot_test;' -U postgres
```

Comment:
I thought we have to create a database in the before script rather than dropping it

Reply:
I missed a line for creating It. I am adding it.